blogposts/2021/release-3.31.md

```diff
@@ -0,0 +1,77 @@
+---
+title: 'Sourcegraph 3.31 release'
+publishDate: 2021-09-01T10:00-07:00
+description: 'Sourcegraph 3.31 introduces the Private Beta release of Code Insights, complete with no repository limits plus repository-based filters. It also introduces automated migration of saved search notifications to code monitoring.'
```

Comment:
Given that we're introducing code insights, most folks won't know there were limits and it brings into question the quality of what we're launching. I don't think the repo limits and repo based filters are helpful at the top level description.

```suggestion
description: 'Sourcegraph 3.31 introduces the Private Beta release of Code Insights and the ability to automatically migrate saved search notifications to code monitors.'
```